new_images.yml

```diff
@@ -1,7 +1,7 @@
 ---
 new_images:
-  - spark: "3.0.0"
+  - spark: "3.1.1"
     use-case: "processing"
     processors: ["cpu"]
-    python: ["py37"]
-    sm_version: "1.4"
+    python: ["py39"]
+    sm_version: "1.0"
```

Comment:
We are downgrading sagemaker version, is this expected?

Reply:
yeah so 1.4 and 1.0 are only part of the tag, because we are releasing 3.1-cpu-py39 for the first time, I set sm_version to 1.0

Reply:
I see, that make sense